funds-manager/funds-manager-server/src/main.rs

```diff
@@ -342,13 +342,19 @@ async fn main() -> Result<(), Box<dyn Error>> {
 
     let create_hot_wallet = warp::post()
         .and(warp::path("hot-wallets"))
-        .and(warp::path(CREATE_HOT_WALLET_ROUTE))
         .and(with_hmac_auth(server.clone()))
         .map(with_json_body::<CreateHotWalletRequest>)
         .and_then(identity)
         .and(with_server(server.clone()))
         .and_then(create_hot_wallet_handler);
 
+    let get_hot_wallet_balances = warp::get()
+        .and(warp::path("hot-wallets"))
+        .and(with_hmac_auth(server.clone()))
```

Comment:
better to be on the safe side, but does this endpoint need hmac auth? i thought the idea was just to protect the "write" / post endpoints

Reply:
without hmac auth anyone would be able to view our balances, that's certainly not okay